app/dependencies.php

```diff
@@ -291,6 +292,10 @@ function (ContainerInterface $c): Auth\DonationPublicAuthMiddleware {
             ]);
         },
 
+        StripeClientInterface::class => static function (ContainerInterface $c): StripeClientInterface {
+            return $c->get(StripeClient::class);
+        },
```

Comment:
@bdsl I think all our dependency injections are written like this, so I think it should be fine.

Reply:
OK, will leave as is. It's just because this is effectively making an alias for an existing service I thought there might be a dedicated syntax to do that.